### PR TITLE
fix(redfish-config): Get redfish host from ipmitool

### PR DESCRIPTION
**Pull request by jneo8**
- The redfish host will be different for each machine and it shouldn't be provided by juju config
- Force user to use https protocol for RedFish.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -17,11 +17,6 @@ options:
       default, it will set to INFO. Allowed values are "DEBUG", "INFO",
       "WARNING", "ERROR", "CRITICAL". Values other than those will result in
       failure of the exporter.
-  redfish-host:
-    type: string
-    default: "http://127.0.0.1"
-    description: |
-      BMC hostname to be used by the redfish collector.
```
Comment (sudeephb) [inline config.yaml:24]: Can't we leave the option for the user to configure the host in case they want?

Comment (jneo8) [reply config.yaml:24]: No, this option is useless in multiple machines environment.

File: src/service.py [hunk 2]
```diff
@@ -89,9 +89,9 @@ def render_config(self, port: str, level: str, redfish_creds: dict) -> bool:
             PORT=port,
             LEVEL=level,
             COLLECTORS=collectors,
-            REDFISH_HOST=redfish_creds["host"],
-            REDFISH_USERNAME=redfish_creds["username"],
-            REDFISH_PASSWORD=redfish_creds["password"],
+            REDFISH_HOST=redfish_creds.get("host"),
```
Comment (Pjack) [inline src/service.py:92]: Any reason that it's better to use `get` function?

**Review by Pjack — APPROVED** [review]

File: src/service.py (hunk 2 above)
Comment (jneo8) [reply src/service.py:92]: It's empty dictionary if the redfish verifier is not pass.

**Review by sudeephb — APPROVED** [review]

**Review by esunar — APPROVED** [review]

**Pull request merged**